package.json

```diff
@@ -43,7 +43,7 @@
 		"js-yaml": "^3.13.1",
 		"limitation": "^0.2.3",
 		"lodash.clonedeep": "^4.5.0",
-		"prom-client": "^11.5.3",
+		"prom-client": "^12.0.0",
```

Comment:
The latest version is 15.1.0. Should we be bold and aim for that?

Reply:
There are a bunch of other breaking changes, and no tests, so I was trying to be conservative in the upgrade.